### PR TITLE
[PS] Componentize Cell #4083

### PR DESCRIPTION
**Pull request by brianacnguyen**
**Description**

#4083 

**Screenshots/Recordings**

Cell Container Select Item
Screen Shot 2022-08-01 at 9 48 10 PM
Cell Container MultiSelect Item
Screen Shot 2022-08-01 at 9 47 55 PM

Cell
Screen Shot 2022-08-01 at 9 47 03 PM
Screen Shot 2022-08-01 at 9 47 25 PM

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Comment by brianacnguyen** [comment]
I have read the CLA Document and I hereby sign the CLA

File: app/component-library/components/Cell/Cell.styles.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    base: Object.assign({} as ViewStyle, style) as ViewStyle,
+    cell: {
+      flexDirection: 'row'
+    },
+    accountAvatar: {
+      marginRight: 16,
+    },
+    cellInfo: {
+      flex: 1,
+      alignItems: 'flex-start'
```
Comment (Cal-L) [inline app/component-library/components/Cell/Cell.styles.ts:31]: Double check if this is needed

File: app/component-library/components/Cell/Cell.styles.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    title: {
+    },
+    secondaryText: {
+      color: colors.text.alternative,
+    },
+    tertiaryText: {
+      color: colors.text.alternative,
+    },
+    label: {
```
Comment (Cal-L) [inline app/component-library/components/Cell/Cell.styles.ts:44]:
```suggestion
    tagLabel: {
```

File: app/component-library/components/Cell/Cell.styles.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      marginRight: 16,
+    },
+    cellInfo: {
+      flex: 1,
+      alignItems: 'flex-start'
+    },
+    optionalAccessory: {
+      marginLeft: 16,
+    },
+    title: {
```
Comment (Cal-L) [inline app/component-library/components/Cell/Cell.styles.ts:36]:
```suggestion
```

File: app/component-library/components/Cell/Cell.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+// 3rd party dependencies
+import React from 'react';
+import { Alert } from 'react-native';
+import { boolean, text } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react-native';
+
+// Base dependencies
```
Comment (Cal-L) [inline app/component-library/components/Cell/Cell.stories.tsx:8]: Separate out dependencies

File: app/component-library/components/Cell/Cell.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  .add('Default', () => {
+    const groupId = 'Props';
+    const isMultiSelect = boolean('IsMultiSelect?', false, groupId);
+    const titleText = text('title', TEST_CELL_TITLE, groupId);
+    const includeSecondaryText = boolean('Includes secondaryText?', false, groupId);
+    const secondaryText = includeSecondaryText ? text('secondaryText', TEST_CELL_SECONDARY_TEXT, groupId) : '';
+    const includeTertiaryText = boolean('Includes tertiaryText?', false, groupId);
+    const tertiaryText = includeTertiaryText ? text('tertiaryText', TEST_CELL_TERTIARY_TEXT, groupId) : '';
+    const includeLabel = boolean('Includes label?', false, groupId);
+    const label = includeLabel ? text('label', TEST_LABEL_TEXT, groupId) : '';
```
Comment (Cal-L) [inline app/component-library/components/Cell/Cell.stories.tsx:29]:
```suggestion
    const tagLabel = includeTagLabel ? text('tagLabel', TEST__TAG_LABEL_TEXT, groupId) : '';
```

File: app/component-library/components/Cell/Cell.constants.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+export const TEST_ACCOUNT_ADDRESS =
+  '0x2990079bcdEe240329a520d2444386FC119da21a';
+export const TEST_CELL_TITLE = 'Orangefox.eth';
+export const TEST_CELL_SECONDARY_TEXT = '0x2990079bcdEe240329a520d2444386FC119da21a';
+export const TEST_CELL_TERTIARY_TEXT = 'Updated 1 sec ago';
+export const TEST_LABEL_TEXT = 'Imported';
```
Comment (Cal-L) [inline app/component-library/components/Cell/Cell.constants.ts:6]:
```suggestion
export const TEST_TAG_LABEL_TEXT = 'Imported';
```

File: app/component-library/components/Cell/Cell.snapshot.test.tsx [hunk 7]
```diff
@@ -0,0 +1,91 @@
+// 3rd party dependencies
+import React from 'react';
+import { shallow } from 'enzyme';
+
+// External dependencies
+import { AccountAvatarType } from '../AccountAvatar';
+
+// Internal dependencies
```
Comment (Cal-L) [inline app/component-library/components/Cell/Cell.snapshot.test.tsx:8]: Move these into unit test file

**Review by Cal-L — APPROVED** [review]
Left some comments!

**Review by brianacnguyen — COMMENTED** [review]
Resolved comments

**Pull request merged**